src/components/AvatarWithIndicator.tsx

```diff
@@ -1,45 +1,41 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import {View} from 'react-native';
 import * as UserUtils from '@libs/UserUtils';
+import {AvatarSource} from '@libs/UserUtils';
 import styles from '@styles/styles';
 import Avatar from './Avatar';
 import * as Expensicons from './Icon/Expensicons';
 import Indicator from './Indicator';
 import Tooltip from './Tooltip';
 
-const propTypes = {
+type AvatarWithIndicatorProps = {
     /** URL for the avatar */
-    source: PropTypes.oneOfType([PropTypes.string, PropTypes.func]).isRequired,
+    source: AvatarSource;
 
     /** To show a tooltip on hover */
-    tooltipText: PropTypes.string,
+    tooltipText?: string;
 
     /** A fallback avatar icon to display when there is an error on loading avatar from remote URL. */
-    fallbackIcon: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+    fallbackIcon?: AvatarSource;
 };
 
-const defaultProps = {
-    tooltipText: '',
-    fallbackIcon: Expensicons.FallbackAvatar,
-};
-
-function AvatarWithIndicator(props) {
+function AvatarWithIndicator({source, tooltipText = '', fallbackIcon = Expensicons.FallbackAvatar}: AvatarWithIndicatorProps) {
     return (
-        <Tooltip text={props.tooltipText}>
+        <Tooltip
+            text={tooltipText}
+            shouldRender={false}
```

Comment:
Why do we even wrap everything in Tooltip if we're never going to render the Tooltip, just its children? Is this actually needed? If so, I think we should explain why